opl/incquery.py

```diff
@@ -182,7 +182,7 @@ def execute(self, name: str, patterns: Hash={}, bindings: Row={}, w_url_provider
             'queryLanguage': 'viatra',
             'queryName': si_query,
             'queryDefinitions': _dict_to_query_defs(h_patterns),
-            'parameterBinding': _dict_to_bindings(h_bindings) if len(h_bindings) != 0 else None
+            'parameterBinding': _dict_to_bindings(h_bindings) if len(h_bindings) != 0 else []
```

Comment:
If the user provided no binding, iqs requires an empty list (rather than None - this would error out)

Reply:
this was fixed in 75d3b49 - can you merge that commit into your pr's branch?